ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Debian.yml

```diff
@@ -90,9 +90,12 @@ Additional_Build_Tools_s390x:
 Additional_Build_Tools_aarch64:
   - libpng-dev
 
+Additional_Build_Tools_riscv64:
+  - locales
```

Comment:
Is this not done by https://github.com/adoptium/infrastructure/blob/793eac15bf0cb46ec92f67d83e6537d7ca9feac5/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml#L196 ? Seems odd to have this as a RISC-V specific clause ... I feel the same is true of systemd-timesyncd

Reply:
`locales` is, from the task you linked ... Not sure about `systemd-timesyncd` ? The `NTP_Time` role doesn't install it

Reply:
I've removed `locales`, however I can't see anywhere else that `systemd-timesyncd` is installed. I took this particular piece of code from @gdams' #1483 , so `systemd-timesyncd` may have been required when setting up the `riscv` machines.